src/components/Navbar/SettingsDrawer/TranslationSection.tsx

```diff
@@ -49,25 +47,41 @@ const TranslationSection = () => {
 
   const renderTranslations = useCallback(
     (data: TranslationsResponse) => {
+      const firstSelectedTranslation = data.translations.find(
+        (translation) => translation.id === selectedTranslations[0],
+      );
+
+      let selectedValueString = t('settings.no-tafsir-selected');
+      if (selectedTranslations.length === 1) selectedValueString = firstSelectedTranslation.name;
+      if (selectedTranslations.length >= 1)
+        selectedValueString = t('settings.value-and-others', {
+          value: firstSelectedTranslation.name,
+          othersCount: selectedTranslations.length - 1,
+        });
+
       return (
-        <div>
-          {selectedTranslations.map((translationId) => {
-            const selectedTranslation = data.translations.find(
-              (translation) => translation.id === translationId,
-            );
-            return <div>{selectedTranslation.name}</div>;
-          })}
-        </div>
+        <SelectionCard
+          label="Selected Translations"
+          value={selectedValueString}
+          onClick={() => dispatch(setSettingsView(SettingsView.Translation))}
+        />
       );
     },
     // eslint-disable-next-line react-hooks/exhaustive-deps
-    [selectSelectedTranslations.length],
+    [selectedTranslations.length],
```

Comment:
same as `selectedTafsirs`, we are using the value so we should put it as a dependency